src/backend/common/MemoryManager.hpp

```diff
@@ -28,132 +31,127 @@ namespace spdlog {
 class logger;
 }
 namespace common {
-using mutex_t      = std::mutex;
+using mutex_t = std::mutex;
 using lock_guard_t = std::lock_guard<mutex_t>;
 
 constexpr unsigned MAX_BUFFERS = 1000;
-constexpr size_t ONE_GB        = 1 << 30;
-
-struct MemoryEventPair {
-    void *ptr;
-    detail::Event e;
-    MemoryEventPair(MemoryEventPair &other)  = delete;
-    MemoryEventPair(MemoryEventPair &&other) = default;
-    MemoryEventPair &operator=(MemoryEventPair &&other) = default;
-    MemoryEventPair &operator=(MemoryEventPair &other) = delete;
+constexpr size_t ONE_GB = 1 << 30;
```

Comment:
Other than moving memory event pair to `src/api/c/` level, rest of the changes are style changes right ? or Am I missing something.

Reply:
This is a clang format issue - sorry. Will fix.